tests/Tokenizer/Transformer/TypeIntersectionTransformerTest.php

```diff
@@ -63,7 +63,7 @@ function foo(){}
             ',
         ];
 
-        if (\PHP_VERSION_ID >= 8_01_00) {
+        if (\defined('T_AMPERSAND_FOLLOWED_BY_VAR_OR_VARARG')) { // @TODO: drop condition when PHP 8.1+ is required
```

Comment:
-1, please replace to dataprovider for 8.1.
very same reasoning. I think I already shared that ;)
https://github.com/PHP-CS-Fixer/PHP-CS-Fixer/pull/7580#discussion_r1428878652

Reply:
And I've already shared I can't. If you can, maybe can you demonstrate it?